frontend/apps/core-app/src/screens/RecipientRegistrationScreen/RecipientRegistrationScreen.container.tsx

```diff
@@ -1,11 +1,42 @@
 import * as React from 'react';
-import { RouteProp, useRoute } from '@react-navigation/native';
-
-import { RootParamList } from '../../navigation/types';
+import { FormDefinition, Record } from 'core-api-client';
 
+import { buildDefaultRecord } from './buildDefaultRecord';
 import { RecipientRegistrationScreenComponent } from './RecipientRegistrationScreen.component';
 
+const getForms = async (): Promise<FormDefinition[]> => Promise.resolve([]);
+
 export const RecipientRegistrationScreenContainer: React.FC = () => {
-  const route = useRoute<RouteProp<RootParamList, 'RecipientRegistration'>>();
-  return <RecipientRegistrationScreenComponent route={route} />;
+  const [mode, setMode] = React.useState<'register' | 'review'>('register');
+  const [forms, setForms] = React.useState<FormDefinition[]>([]);
+  const [records, setRecords] = React.useState<Record[]>([]);
+
+  React.useEffect(() => {
+    (async () => {
+      const formsResponse = await getForms();
+      setForms(formsResponse);
+    })();
+  }, []);
+
+  React.useEffect(() => {
+    setRecords(forms.map(buildDefaultRecord));
+    setMode('register');
```

Comment:
is this necessary when it's the default?

Reply:
I'm not sure what you mean

Reply:
mode is by default set to register, and I don't see any point where it could be set to review, so this didn't seem necessary. but the code setting mode to review will come later, and then this will make sense, so nevermind.